docs/source/x_ext.rst

```diff
@@ -51,6 +51,8 @@ Parameters
 +------------------------------+------------------------+---------------+--------------------------------------------------------------------+
 | ``X_MISA``                   | logic [31:0]           | 0x0000_0000   | MISA extensions implemented on the eXtension interface.            |
 +------------------------------+------------------------+---------------+--------------------------------------------------------------------+
+| ``X_ECS_XS``                 | logic [1:0]            | 2'b0          | Default value for ``mstatus.vs``.                                  |
++------------------------------+------------------------+---------------+--------------------------------------------------------------------+
```

Comment:
Is this just a typo, or does the signal X_ECS_**_XS_** really correspond to mstatus.**_vs_**?

If it is not a typo, then I would suggest changing the parameter name to X_ECS_**_VS_**

Reply:
That is a typo indeed. However, the parameter name is right and the signal name is wrong. the XS field is special in the sense that it is read only and considered to track some externally implemented status bit (and therefore we need a way to set the initial status of the XS bitfield in the core according to how the corresponding bits are initialized inside a coprocessor).